packages/plugin-webgazer-validate/src/index.ts

```diff
@@ -1,18 +1,14 @@
-/**
- * jspsych-webgazer-validate
- * Josh de Leeuw
- **/
+import { JsPsych, JsPsychPlugin, ParameterType, TrialType } from "jspsych";
 
-import jsPsych from "jspsych";
+// TO DO: the parameters below don't match up with the docs: https://www.jspsych.org/plugins/jspsych-webgazer-validate/
+// docs contain 'repetiton_per_point', and some param descriptions in docs refer to calibration
 
-const plugin = <any>{};
-
-plugin.info = {
+const info = <const>{
   name: "webgazer-validate",
-  description: "",
   parameters: {
+    /* Array of points in [x,y] coordinates */
     validation_points: {
-      type: jsPsych.plugins.parameterType.INT,
+      type: ParameterType.INT,
```

Comment:
Yes, also true for `calibration_points` parameter in `webgazer-calibrate` plugin, and `grid` parameters in `serial-reaction-time` and `serial-reaction-time-mouse` plugins. Left this unchanged for now.

Reply:
@bjoluc @jodeleeuw any suggestions for what to do for parameter types that are arrays of arrays?

Reply:
I think `ParameterType.ANY` would be a useful addition for all the edge cases that may exist, including this one. Alternatively, one could think about a "complex"-like array type, but I don't think the effort would be worth it.

Reply:
Thanks @bjoluc, I've been wanting to tidy up these last few parameter type edge cases but wasn't sure how. @jodeleeuw if you're happy with adding an `ANY` type then maybe I can do that on the `modularization` branch, since this is already merged?